src/LowLevel/SecurityToken.ts

```diff
@@ -205,6 +237,26 @@ export class SecurityToken extends Contract<SecurityTokenContract> {
     return addresses.map(address => new UsdTieredSto({ address, context }));
   }
 
+  /**
+   * Given STO module type and address, this function will return a generic, LowLevel Sto object.
+   *
+   * @TODO remon-nashid: shouldn't module address be enough to fetch a module?
```

Comment:
Address should be enough, but you need to return the appropriate LowLevel class, and for that you need the type